lib/php/rendition-factory/src/Transformer/Image/Imagine/ImagineTransformerModule.php

```diff
@@ -32,21 +32,302 @@ public function getDocumentation(): Documentation
+                                    ->end()
+                                ->end()
+                            ->end()
+                            ->arrayNode('crop')
+                                ->info('filter performs sizing transformations (which includes cropping operations)')
+                                ->children()
+                                    ->arrayNode('size')
+                                        ->info('set the size of the cropping area [width,height]')
+                                        ->children()
+                                            ->integerNode(0)
+                                                ->example(300)
+                                            ->end()
+                                            ->integerNode(1)
+                                                ->example(600)
+                                            ->end()
+                                        ->end()
+                                    ->end()  
+                                    ->arrayNode('start')
+                                        ->info('Sets the top, left-post anchor coordinates where the crop operation starts[x, y]')
+                                        ->children()
+                                            ->integerNode(0)
+                                                ->example(040)
+                                            ->end()
+                                            ->integerNode(1)
+                                                ->example(160)
+                                            ->end()
+                                        ->end()
+                                    ->end()  
+                                ->end()
+                            ->end()
+                            ->arrayNode('watermark')
+                                ->info('filter adds a watermark to an existing image')
+                                ->children()
+                                    ->scalarNode('image')
+                                        ->info('Path to the watermark image')
+                                    ->end()
+                                ->end()
+                            ->end()
+                            ->arrayNode('background_fill')
+                                ->info('filter fill background color')
+                                ->children()
+                                    ->scalarNode('color')
+                                        ->info('Sets the background color HEX value. The default color is white (#fff).')
+                                        ->end()
+                                    ->integerNode('opacity')
+                                        ->info('Sets the background alpha value. The value should be within a range of 0 (opaque) - 100 (fully transparent). default opacity 100')
```

Comment:
this is the opposite : 0% opacity = transparent ; 100% opacity=opaque
```suggestion
                                        ->info('Sets the background alpha value. The value should be within a range of 0 (opaque) - 100 (fully transparent). default opacity 100')
```